ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -332,9 +364,20 @@ const AssetListControlBar = ({
           }
           color={TextColor.textDefault}
           marginRight={isFullScreen ? 2 : null}
+          borderColor={BorderColor.borderDefault}
           ellipsis
         >
-          {networkButtonText}
+          <Box display={Display.Flex} alignItems={AlignItems.center} gap={2}>
+            {singleNetworkIconUrl && (
+              <AvatarNetwork
+                name={currentMultichainNetwork.nickname}
+                src={singleNetworkIconUrl}
+                size={AvatarNetworkSize.Sm}
+                borderWidth={0}
+              />
+            )}
+            <Text ellipsis>{networkButtonText}</Text>
+          </Box>
```

Comment:
optional image for when a single network is selected.
Ensured that the text ellipsis still functions if network name is very large.

Reply:
<img width="622" height="85" alt="Screenshot 2025-08-22 at 20 54 48" src="https://github.com/user-attachments/assets/4c78d375-40ac-4e9e-8434-64ce0887b6e6" />